### DISCUSSION TITLE
Qual o empacotador mais recomendado? Webpack ou Browserify?

### DISCUSSION THREAD

**Discussion by israelmarmar** [frontendbr/forum#997] [Perguntas e Respostas]

**Comment by fdaciuk** [comment]
Hoje acredito que o webpack é o mais utilizado, mas além desses dois, ainda tem boas opções como Rollup (mais usado para libs), e o Parcel, que veio com a proposta de funcionar com zero config.

Hoje o webpack4 já tem essa opção de funcionar com zero config, muito por conta do que o Parcel trouxe =)

Enfim, tem várias opções que você pode usar. Teste algumas e use a que achar melhor =)

**Comment by woliveiras** [comment]
Seria legal comentar um pouco mais sobre o seu objetivo, @israelmarmar.

Para o que vai utilizar o empacotador, se irá utilizar algum framework ou lib e coisas do tipo, viu. :)

**Comment by alexsandro-xpt** [comment]
O Webpack é bem completo. Uso desde a versão 1.x. Lembro que quando comecei achei muito verboso e na primeira impressão achei que iria ser muito difícil comparando o RequireJS que eu usava.

Neste meio tempo pensei em usar o Browserify, mas eu insisti do Browserify com ideia de que o Webpack era melhor.
Em uma semana eu pude focar e estudei ele e vi que foi o suficiente pra entender 90% dele, apesar de ter muitas configurações era bem fácil de aprender e gostar. Para mim foi intuitivo.

**Comment by giovannism20** [comment]
O @tiaguinho escreveu um artigo na @braziljs sobre um outro empacotador : 

https://braziljs.org/blog/compactacao-maxima-com-brotli/

**Comment by willycamargo** [comment]
A thread é sobre empacotador e não sobre compressor, @GiovanniSM20 😄 
Mas valeu pela informação.

**Comment by ghost** [comment]
Recomendo o webpack

**Comment by mahenrique94** [comment]
Webpack +1

**Comment by ramonsenadev** [comment]
Utilizo o webpack, mas recomendo testar os outros e utilizar o que deixar seu processo de desenvolvimento mais confortável. 💃
